packages/docs/stories/components/input.stories.ts

```diff
@@ -156,42 +155,51 @@ export const Invalid: Story = {
   },
   play: async ({ canvasElement }) => {
     try {
-      const input = canvasElement.querySelector('syn-input');
-      const button = canvasElement.querySelector('button');
+      const form = canvasElement.querySelector('form');
+
+      if (!form) {
+        return;
+      }
+
+      const input = form.querySelector('syn-input') as SynInput;
+      const button = form.querySelector('syn-button') as SynButton;
 
-      await waitUntil(() => input?.shadowRoot?.querySelector('input'));
+      await waitUntil(() => Promise.allSettled([
+        customElements.whenDefined('syn-input'),
+        customElements.whenDefined('syn-button'),
+      ]));
```

Comment:
We had some issues in Solid with whenDefined or similar because sometimes the element wasn't completely rendered. But yah, maybe it works now, let's see.

Reply:
Interesting, I can see this in Chromatic, but not on Storybook itself. Will have to check why the Screenshots look scrambled.

Reply:
What's your plan on this? As this ticket would be important to show our real buttons (design already complained) I think we should stick to the initial task and remove these changes to get it done.

Reply:
Just pushed a new version of this branch where I also updated all components to use syn-button in the play functions. The calls to waitUntil are removed there as well, seems to work fine (at least locally)